ldes-server-infra-postgres/postgres-admin-repository/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/postgres/eventstream/v2/entity/EventStreamEntity.java

```diff
@@ -0,0 +1,58 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.admin.postgres.eventstream.v2.entity;
+
+import jakarta.persistence.*;
+
+@Entity(name = "collections")
+@Table(name = "collections")
+public class EventStreamEntity {
```

Comment:
I'd put an index on the name. Although this table won't get big, it's good to index a column that will be heavily queried on

Reply:
An index on name is created by 
`@Column(unique = true, ...)`